javascript/webapp/src/components/ComplexTableComponent/ComplexTableComponent.tsx

```diff
@@ -33,7 +33,7 @@ export const ComplexTableComponent: React.FC<{
 			<Tables>
 				{dataObject.data?.map((row, ridx) => {
 					const id = row[0] as string
-					const graph = relatedGraphs.get(id)
+					const graph = relatedGraphs !== undefined ? relatedGraphs.get(id): undefined
```

Comment:
this could be simplified with `relatedGraphs?.get(id)` instead of the undefined check. the ? is the shortcut